mysql/lib/dialect/mysql2_driver_dialect.ts

```diff
@@ -26,6 +26,7 @@ import { HostInfo } from "../../../common/lib/host_info";
 
 export class MySQL2DriverDialect implements DriverDialect {
   protected dialectName: string = this.constructor.name;
+  keepAlivePropertyNames: string[] = [];
```

Comment:
Is there a reason this is not a static constant? can this be modified?

Reply:
Why the type is string[], not just string?

Reply:
It's a string[] because there could be multiple properties, for example PG having both keepalive and keepaliveInitialDelayMillis